index.html

```diff
@@ -180,11 +226,13 @@ <h2>Skills</h2>
         </div>
       </div>
     </section>
-    <section class="contact">
-      <p>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </p>
+    <section class="contact" id="contact-link">
+      <div class="last-p">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
```

Comment:
- [ ] As requested by the previous, kindly ensure that you stick to the design as much as possible to make your project look good.

your design:
![Screenshot from 2023-01-19 16-31-03](https://user-images.githubusercontent.com/90524466/213469405-b366d8b8-7f23-4c73-94f5-582da7f04f6f.png)

Figma design:
![Screenshot from 2023-01-19 16-36-35](https://user-images.githubusercontent.com/90524466/213470340-52f225ce-0419-4f66-8a50-306788bb23bc.png)